<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmployerWindow</class>
 <widget class="QDialog" name="EmployerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Бюро трудоустройства</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>0</y>
     <width>1200</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1200</width>
     <height>700</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Список вакансий</string>
    </attribute>
    <widget class="QTableWidget" name="vacancyTableWidget">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>1170</width>
       <height>550</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1170</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1170</width>
       <height>550</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="DeleteVacancyButton">
     <property name="geometry">
      <rect>
       <x>511</x>
       <y>610</y>
       <width>175</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Удалить вакансию</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Создать вакансию</string>
    </attribute>
    <widget class="QPushButton" name="CreateVacancyButton">
     <property name="geometry">
      <rect>
       <x>511</x>
       <y>610</y>
       <width>175</width>
       <height>35</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>175</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>Создать</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>9</y>
       <width>1161</width>
       <height>508</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>350</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Должность</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="TitleEdit">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>35</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Оклад</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="SalaryEdit">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Срок</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PeriodEdit">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Требования</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RequirementsEdit">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Дополнительная информация</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="AdditionalDataEdit">
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>175</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>175</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Посмотреть отклики</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_2">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>1170</width>
       <height>550</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1170</width>
       <height>550</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID Вакансии</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Профессия</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Квалификация</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Доп. Данные</string>
      </property>
     </column>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>511</x>
       <y>610</y>
       <width>175</width>
       <height>35</height>
      </rect>
     </property>
     <property name="text">
      <string>Заключить договор</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Совершенные сделки</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_3">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>30</y>
       <width>1170</width>
       <height>550</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1170</width>
       <height>550</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1170</width>
       <height>550</height>
      </size>
     </property>
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ФИО</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Должность</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Зарплата</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Период</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Комиссионые бюро</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
